src/widgets/auto-styler.js

```diff
@@ -0,0 +1,141 @@
+var cdb = require('cartodb.js');
```

Comment:
This class deserves a unit test... There's a lot of "meat" inside which is untested.